include/circt/Dialect/FIRRTL/FIRRTLDialect.h

```diff
@@ -22,9 +22,11 @@ namespace firrtl {
 
 class FIRRTLType;
 
-/// If the specified attribute list has a firrtl.name attribute, return its
-/// value.
-StringAttr getFIRRTLNameAttr(ArrayRef<NamedAttribute> attrs);
+const StringRef portNameAttrStringRef("portNames");
+
+/// Return the portNames attribute for the specified module, which contains the
+/// name for each port.
+ArrayAttr getModulePortNames(Operation *module);
```

Comment:
Thanks for the rename, plz move this to FIRRTLOps.h to be next to getModulePortName.